tt-metal-yolov4/server/fast_api_yolov4.py

```diff
@@ -0,0 +1,217 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+import os
+import logging
+from fastapi import FastAPI, File, UploadFile
+from io import BytesIO
+from PIL import Image
+from models.demos.yolov4.tests.yolov4_perfomant_webdemo import Yolov4Trace2CQ
+import ttnn
+
+import numpy as np
+import torch
+import time
+
+app = FastAPI(
+    title="YOLOv4 object detection",
+    description="Inference engine to detect objects in image.",
+    version="0.0",
+)
+
+
+@app.get("/")
+async def root():
+    return {"message": "Hello World"}
+
+
+# Configure the logger
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s - %(levelname)s - %(message)s",
+    handlers=[logging.StreamHandler()],
+)
+
+
+def get_dispatch_core_type():
+    # TODO: 11059 move dispatch_core_type to device_params when all tests are updated to not use WH_ARCH_YAML env flag
+    dispatch_core_type = ttnn.device.DispatchCoreType.WORKER
+    # if ("WH_ARCH_YAML" in os.environ) and os.environ["WH_ARCH_YAML"] == "wormhole_b0_80_arch_eth_dispatch.yaml":
```

Comment:
Do we need this comment here?

Reply:
No, removed in https://github.com/tenstorrent/tt-inference-server/pull/52/commits/be8d0bf52fd56c13b5b718cdee7a4f43042a9796